frontend/SearchPane.js

```diff
@@ -53,14 +53,6 @@ class SearchPane extends React.Component {
   }
 
   onDocumentKeyDown(e) {
-    if (e.keyCode === 191) { // forward slash
-      var doc = ReactDOM.findDOMNode(this).ownerDocument;
-      if (!this.input || doc.activeElement === this.input) {
-        return;
-      }
```

Comment:
I don't know that codebase very well, but maybe you could check on if `doc.activeElement` is an input field and `return` if that's the case? That would leave the functionality intact for the most part.

That's assuming that the prop/state field is actually an input field, but, if not, some other similar test can probably be performed.